cmd/embedded-cluster/install.go

```diff
@@ -473,21 +509,21 @@ var installCommand = &cli.Command{
 			metrics.ReportApplyFinished(c, err)
 			return err
 		}
-		logrus.Debugf("running host preflights")
-		if err := RunHostPreflights(c); err != nil {
-			err := fmt.Errorf("unable to finish preflight checks: %w", err)
-			metrics.ReportApplyFinished(c, err)
-			return err
-		}
 		logrus.Debugf("creating k0s configuration file")
 		if err := ensureK0sConfig(c); err != nil {
 			err := fmt.Errorf("unable to create config file: %w", err)
 			metrics.ReportApplyFinished(c, err)
 			return err
 		}
+		logrus.Debugf("running host preflights")
```

Comment:
Isn't this, assuming preflights fail, force a `./embedded-cluster reset` before attempting to install again ?

Reply:
If rerunning `install` after host preflights fail is totally fine, then let's not tell them to reset. We should only force a reset if the install already did something that could cause an issue down the line.